.eslintrc.json

```diff
@@ -29,6 +29,7 @@
         "tsx": "never"
       }
     ],
+    "import/prefer-default-export": "off",
```

Comment:
While it might make sense in some places, in others (like `ui.ts` files) it causes inconsistencies, so I opted for turning it off. There are projects where 0 default exports are used, so it's really just a matter of preference.